### PR TITLE
Ekapner readme updates

### PR DESCRIPTION
**Pull request by ekapner**
### What does this PR do?
Updates the 3 project:create readmes for analytics, standard, empty
Reduce maintenance and overhead by removing duplicate documentation

### What issues does this PR fix or reference?
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008g3ZhIAI/view

### CODE REVIEW

File: packages/templates/src/templates/project/analytics/README.md [hunk 1]
```diff
@@ -1,59 +1,15 @@
-# Salesforce App
+# Salesforce DX Project: Next Steps
```
Comment (smithgp) [inline packages/templates/src/templates/project/analytics/README.md:1]: @mblumreich Mandy, this is the README that gets laid down when someone creates an analytics project from sfdx (or from vscode). We might to add some analytics-specific text and/or links here at some point.
@ekapner The base text here looks good to me, thanks for cleaning it up!

Comment (ekapner) [reply packages/templates/src/templates/project/analytics/README.md:1]: @mblumreich - my main goal was to reduce maintenance and overhead of duplicating content in multiple places.

**Review by mblumreich — APPROVED** [review]
This looks great! @smithgp let's brainstorm at some point on Analytics-specific content, but with the goal to keep it light

**Review by lcampos — APPROVED** [review]
aside from the changes to the analytics readme the rest looks good

**Pull request merged**